### PR TITLE
update Ant version info in readme

### PR DESCRIPTION
**Pull request by SethTisue**

### CODE REVIEW

**Comment by SethTisue** [comment]
as discovered by Adriaan today in the course of some highly scintillating Chef work

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ You need the following tools:
   - sbt, we recommend the [sbt-extras](https://github.com/paulp/sbt-extras) runner
   script. It provides sensible default jvm options (stack and heap size).
   - curl (for `./pull-binary-libs.sh`, used by the sbt / ant build).
-  - Apache Ant (version 1.9.3 or above) if you need to use the (deprecated) ant build.
+  - Apache Ant (version 1.9.x, minimum 1.9.3; Ant 1.10+ doesn't work on Java 6) if you are using the ant build.
```
Comment (janekdb) [inline README.md:74]: Easier to decide from?

* Apache Ant if your are using the ant build.
  * Version >= 1.9.3 if using Java 8
  * Version >= 1.9.3, < 1.10 if using Java 6

Comment (SethTisue) [reply README.md:74]: I'd only want to say that if I'd verified that using Ant 1.10 and Java 8 even works, but I haven't. (JDK 6 being the recommended JDK for building 2.11.)

Comment (janekdb) [reply README.md:74]: yes, good point. I'd actually taken more from your edit than was there so LGTM.

Comment (janekdb) [inline README.md]: LGTM

**Review by janekdb — APPROVED** [review]

**Pull request merged**